Framework/Core/src/ArrowTableSlicingCache.cxx

```diff
@@ -25,6 +25,14 @@ std::pair<int64_t, int64_t> SliceInfoPtr::getSliceFor(int value) const
   if (values.empty()) {
     return {offset, 0};
   }
+  auto p = values.size() - 1;
```

Comment:
```suggestion
  assert(!values.empty());
  auto p = values.size() - 1;
```

Reply:
Non-empty table would never result in an empty values array. This is checked at the stage of cache creation.